UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1443,6 +1416,41 @@ private void FastRegen()
 				part.HealDamage(null, fastRegenHeal, DamageType.Brute);
 			}
 		}
+
+		public void SetUpCharacter(PlayerHealthData RaceBodyparts)
+		{
+			if (CustomNetworkManager.Instance._isServer)
+			{
+				InstantiateAndSetUp(RaceBodyparts.Base.Head);
+				InstantiateAndSetUp(RaceBodyparts.Base.Torso);
+				InstantiateAndSetUp(RaceBodyparts.Base.ArmLeft);
+				InstantiateAndSetUp(RaceBodyparts.Base.ArmRight);
+				InstantiateAndSetUp(RaceBodyparts.Base.LegLeft);
+				InstantiateAndSetUp(RaceBodyparts.Base.LegRight);
+			}
+		}
+
+		public void InitialiseFromRaceData(PlayerHealthData RaceBodyparts)
```

Comment:
How does this handle races that may not have metabolism, blood or toxins?

Reply:
doesn't currently but it is easy enough to add

Reply:
would be good to have